pipx/pipxrc.py

```diff
@@ -0,0 +1,195 @@
+import json
+from pathlib import Path
+from typing import List, Dict, NamedTuple, Any, Optional, TypeVar
+
+from pipx.Venv import PipxVenvMetadata
+
+
+PIPX_INFO_FILENAME = "pipxrc.json"
+
+
+class JsonEncoderPipx(json.JSONEncoder):
+    def default(self, obj):
+        # only handles what json.JSONEncoder doesn't understand by default
+        if isinstance(obj, Path):
+            return {"__type__": "Path", "__Path__": str(obj)}
+        return super().default(obj)
+
+
+def _json_decoder_object_hook(json_dict):
+    if json_dict.get("__type__", None) == "Path" and "__Path__" in json_dict:
+        return Path(json_dict["__Path__"])
+    return json_dict
+
+
+# Used for consistent types of multiple kinds
+Multi = TypeVar("Multi")
+
+
+class InjPkg(NamedTuple):
+    pip_args: List[str]
+    verbose: bool
+    include_apps: bool
+    include_dependencies: bool
+    force: bool
+
+
+class InstallOpts(NamedTuple):
+    pip_args: Optional[List[str]]
+    venv_args: Optional[List[str]]
+    include_dependencies: Optional[bool]
+
+
+class PipxrcInfo:
```

Comment:
I think a better name for this would be `PipxInfo`. `rcInfo` is kind of redundant since they both kind of imply a similar thing, and when I think of `rc` I think of a file a user creates to define preferences on how an application runs rather than metadata associated with a tool. In fact, `PipxPackageMeta` would probably be my vote.

Reply:
Nevermind, I see you already have a class with that name. This name makes sense now!